cmake/all.cmake

```diff
@@ -43,7 +43,11 @@ endif()
 # or CMAKE_PREFIX_PATH from the environment
 if(NOT DEFINED CMAKE_PREFIX_PATH)
   if(NOT "$ENV{CMAKE_PREFIX_PATH}" STREQUAL "")
-    string(REPLACE ":" ";" CMAKE_PREFIX_PATH $ENV{CMAKE_PREFIX_PATH})
+    if(NOT WIN32)
+      string(REPLACE ":" ";" CMAKE_PREFIX_PATH $ENV{CMAKE_PREFIX_PATH})
+    else()
+	  set(CMAKE_PREFIX_PATH $ENV{CMAKE_PREFIX_PATH})
+	endif()
```

Comment:
The indentation in these lines is off. Please update the patch to use only spaces.